tests/unit/amplitude/test_sympy_model.py

```diff
@@ -0,0 +1,10 @@
+from expertsystem.amplitude import generate_sympy
+from expertsystem.reaction import Result
+
+
```

Comment:
We would need a couple of unit tests that validate the correctness of the generated amplitude model. We can try to think of some good tests. Now we also have the possibility to do some mathematical operations like integrals or normal evaluations!

Reply:
I already ported some tests here, that perform the angular distribution checks currently done in tensorwaves. The tests also become much more elegant now